proxy/ProxyClientSession.h

```diff
@@ -69,6 +69,9 @@ struct ProxyError {
   uint32_t code       = 0;
 };
 
+// A little ugly, but this global is tracked by traffic_server.
+extern bool ts_is_draining;
```

Comment:
We should create a naming standard for globals (TS_ or something).

Reply:
Agreed, but I couldn't find anything existing :).